tests/components/climate/test_init.py

```diff
@@ -953,6 +955,65 @@ async def test_no_issue_no_aux_property(
     ) not in caplog.text
 
 
+async def test_humidity_validation(
+    hass: HomeAssistant, register_test_integration: MockConfigEntry
+) -> None:
+    """Test validation for humidity."""
+
+    class MockClimateEntityTemp(MockClimateEntity):
```

Comment:
We're testing humidity here :)
```suggestion
    class MockClimateEntityHumidity(MockClimateEntity):
```